pkgs/development/python-modules/pdoc/default.nix

```diff
@@ -24,7 +24,7 @@ buildPythonPackage rec {
     owner = "mitmproxy";
     repo = "pdoc";
     rev = "v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
> tested with `--check`

Reply:
when you change a fod you have to either `invalidateByDrvPath`, `--check` or `--rebuild` it